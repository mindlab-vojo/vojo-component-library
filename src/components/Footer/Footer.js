import React from 'react'
import PropTypes from 'prop-types'

import * as Footer from './Footer.style'

import ThemeConsumer from '../../style/ThemeManager/ThemeProvider'
import Logo from '../Logo'
import Typography from '../Typography'

function FooterComponent({ termsUrl }) {

  const actualYear = new Date().getFullYear()
  const linkAdditionalProps = { href: termsUrl, target: "_blank" }

  const renderComponent = (theme) => (
    <Footer.Wrapper 
      className="Footer"
      theme={theme}>
      <Footer.Content
        className="Footer_Content"
        theme={theme}>
        <Footer.ContentBlock
          className="Footer_Content__Block"
          theme={theme}>
          <Typography
            additionalProps={linkAdditionalProps}
            color={theme.colors.whiteColor}
            tag="a"
            type="default"
            isUnderlined>
            Termos de uso
          </Typography>
        </Footer.ContentBlock>
        <Footer.ContentBlock
          className="Footer_Content__Block"
          theme={theme}>
          <Typography
            color={theme.colors.whiteColor}
            tag="p"
            type="default">
            {actualYear} – Vojo® – Todos os direitos reservados 
          </Typography>
          <Logo isReverse
            height="16px"
            width="103px"
          />
        </Footer.ContentBlock>
      </Footer.Content>
    </Footer.Wrapper>
  )

  return (
    <ThemeConsumer>
      {({ theme }) => renderComponent(theme)}
    </ThemeConsumer>
  )
}

FooterComponent.propTypes ={
  termsUrl: PropTypes.string
}

export default FooterComponent

