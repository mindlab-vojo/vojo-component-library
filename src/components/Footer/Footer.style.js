import styled from 'styled-components';

export const Wrapper = styled.footer`
  background-color: ${props => props.theme.colors.primaryColor};
  align-items: center;
  bottom: 0;
  display: flex;
  height: 60px;
  justify-content: center;
  position: fixed;
  width: 100%;
`
export const Content = styled.div`
  align-items: center;
  display: flex;
  justify-content: space-between;
  padding: 0 10px;
  max-width: 960px;
  width: 100%;
`

export const ContentBlock = styled.div`
  align-items: center;
  display: flex;
  justify-content:center;

  .Logo {
    margin-left: 10px;
  }
`