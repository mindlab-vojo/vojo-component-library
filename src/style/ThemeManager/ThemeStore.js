import { vojo } from './../themes'

class ThemeStore {

  constructor(theme) {
    this.defaultTheme = vojo
    this.theme = theme ? theme : this.defaultTheme
  }

  configure = ({ theme }) =>  {
    if (theme) this.configureTheme(theme)
    else this.configureTheme(this.defaultTheme)
  }

  configureTheme = (theme) => {
    this.theme = theme
  }

  modify = (modifiedProps) => {
    const currentTheme = this.theme
    return { currentTheme, modifiedProps }
  }

  getTheme = () => this.theme

}

export default ThemeStore