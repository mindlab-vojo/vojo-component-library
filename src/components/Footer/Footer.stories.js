import React from 'react';
import { withKnobs, text } from '@storybook/addon-knobs';
import { withA11y } from '@storybook/addon-a11y';

import Footer from '.'; 
import { ThemeProvider } from "../../../src";
import categories from '../../../.storybook/categories';

import Container from '../Container';

export const component = () => {
  const termsUrl = text('termsUrl', 'https://app.vojo.com.br/terms-of-use')
  
  return (
    <Container>
      <div style={{ height: '1000px' }}/>
      <Footer termsUrl={termsUrl}/>
    </Container>
  )
}

export default {
  title: categories.Footers,
  component: component,
  decorators: [withKnobs, withA11y, (story) => <ThemeProvider>{ story() }</ThemeProvider>]
}

component.story = {
  name: 'Footer',
};
  