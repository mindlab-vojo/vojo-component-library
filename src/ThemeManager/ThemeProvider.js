import React, { Component, createContext } from 'react'
import PropTypes from 'prop-types'

import ThemeStore from './ThemeStore';

// Provider and Consumer instances
const { Provider, Consumer } = createContext();

class ThemeProvider extends Component {
  static propTypes = {
    children: PropTypes.node,
    theme: PropTypes.string
  }

  state = {
    themeConfig: new ThemeStore()
  }

  componentDidMount() {
    if (this.props.theme) this.setTheme()
  }

  componentDidUpdate(prevProps) {
    if (this.props.theme !== prevProps.theme) this.setTheme()
  }

  setTheme = () => {
    const { theme } = this.props
    const { themeConfig } = this.state
    const updatedTheme = themeConfig.configure({ theme })
    this.setState({ theme: updatedTheme })
  }

  render() {
    return (
      <Provider value={{ theme: this.state.themeConfig.theme }}>
        {this.props.children}
      </Provider>
    )
  }
}

export { ThemeProvider }

export default Consumer