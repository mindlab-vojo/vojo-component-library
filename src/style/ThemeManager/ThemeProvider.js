import React, { Component, createContext } from 'react'
import PropTypes from 'prop-types'

import ThemeStore from './ThemeStore';

// Provider and Consumer instances
const { Provider, Consumer } = createContext();

class ThemeProvider extends Component {
  static propTypes = {
    children: PropTypes.node,
    theme: PropTypes.string
  }

  state = {
    themeConfig: new ThemeStore()
  }

  componentDidMount() {
    if (this.props.theme) this.setTheme()
  }

  componentDidUpdate(prevProps) {
    if (this.props.theme !== prevProps.theme) this.setTheme()
  }

  setTheme = () => {
    const { theme } = this.props
    const { themeConfig } = this.state
    const updatedTheme = themeConfig.configure({ theme })
    this.setState({ theme: updatedTheme })
  }

  getThemeFont = () => {
    const { themeConfig } = this.state
    return themeConfig.font
  }

  render() {

    const GlobalFont = this.getThemeFont()

    return (
      <Provider value={{ theme: this.state.themeConfig.theme }}>
        <GlobalFont/>
        {this.props.children}
      </Provider>
    )
  }
}

export { ThemeProvider }

export default Consumer