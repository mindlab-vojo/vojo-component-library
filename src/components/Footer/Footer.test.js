import React from 'react';
import { render } from '@testing-library/react';

import Footer from './';
import { ThemeProvider } from './../../style/ThemeManager/ThemeProvider';

test('Test Footer component render', () => {
  const component = render(
    <ThemeProvider>
      <Footer termsUrl="https://app.vojo.com.br/terms-of-use"/>
    </ThemeProvider>
  );
  expect(component).toMatchSnapshot()
});
