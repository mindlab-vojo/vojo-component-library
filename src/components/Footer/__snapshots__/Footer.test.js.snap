// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-AxjAm fPNbwy"
      >
        <div
          class="sc-AxirZ RVNDS"
        >
          <div
            class="sc-AxiKw dUPrhR"
          >
            <p
              class="sc-fznZeY iAWmBo"
            >
              Siga o Vojo nas redes sociais:
            </p>
            <div
              class="sc-AxhUy dtWeay"
            >
              <img
                src="https://s3.amazonaws.com/static.vojo.com.br/prd/images/facebook_branco.png"
              />
              <img
                src="https://s3.amazonaws.com/static.vojo.com.br/prd/images/instagram_branco.png"
              />
            </div>
          </div>
          <div
            class="sc-AxhCb jSTosF"
          >
            <div
              class="sc-AxgMl lxzaH"
            >
              <span
                class="sc-AxheI kkCIQt"
              >
                Termos de uso
              </span>
              <span
                class="sc-Axmtr bkBkiv"
              >
                Política de Privacidade
              </span>
            </div>
            <div
              class="sc-AxmLO ffTKFl"
            >
              <span
                class="sc-fzozJi cRdDET"
              >
                O VOJO é um produto da
                <span
                  class="sc-fzpans cFTAwv"
                >
                  Mind Lab
                </span>
                . Estamos localizados na Vila Mariana, em São Paulo – SP
                <span
                  class="sc-fzoLsD kAsOlz"
                >
                  <svg
                    fill="#FF5477"
                    height="12px"
                    stroke="#FF5477"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.5"
                    viewBox="0 0 24 24"
                    width="12px"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"
                    />
                  </svg>
                </span>
              </span>
            </div>
          </div>
          <div
            class="sc-fzplWN gdALIP"
          >
            <span
              class="sc-fznyAO iHyRZ"
            >
              2021
               – Vojo® – Todos os direitos reservados
            </span>
            <div
              class="sc-fznKkj jTYmss"
            >
              <svg
                class="Logo"
                height="16px"
                version="1.1"
                viewBox="0 0 618 96"
                width="103px"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              >
                <title>
                  logo-vojo
                </title>
                <g
                  fill="none"
                  fill-rule="evenodd"
                  stroke="none"
                  stroke-width="1"
                >
                  <g>
                    <path
                      d="M601.267148,96 C605.982247,96 609.949831,94.3888756 613.169898,91.1666267 C616.389966,87.9443778 618,83.9165668 618,79.0831935 C618,74.1347399 616.389966,70.1356989 613.169898,67.0860705 C609.949831,64.0364421 605.982247,62.5116279 601.267148,62.5116279 C596.552049,62.5116279 592.584465,64.0364421 589.364398,67.0860705 C586.14433,70.1356989 584.534296,74.1347399 584.534296,79.0831935 C584.534296,83.9165668 586.14433,87.9443778 589.364398,91.1666267 C592.584465,94.3888756 596.552049,96 601.267148,96 Z"
                      fill="#FFFFFF"
                      id="Ponto"
                    />
                    <path
                      d="M209.811372,-5.62749847e-13 C219.49429,-5.62749847e-13 228.200033,2.07734807 235.928601,6.2320442 C243.657169,10.3867403 249.742306,16.1104972 254.184012,23.4033149 C258.625718,30.6961326 260.84657,38.8950276 260.84657,48 C260.84657,57.1049724 258.625718,65.3038674 254.184012,72.5966851 C249.742306,79.8895028 243.657169,85.6132597 235.928601,89.7679558 C228.200033,93.9226519 219.49429,96 209.811372,96 C200.128453,96 191.400502,93.9226519 183.627517,89.7679558 C175.854532,85.6132597 169.769395,79.8895028 165.372106,72.5966851 C160.974818,65.3038674 158.776173,57.1049724 158.776173,48 C158.776173,38.8950276 160.974818,30.6961326 165.372106,23.4033149 C169.769395,16.1104972 175.854532,10.3867403 183.627517,6.2320442 C191.400502,2.07734807 200.128453,-5.62749847e-13 209.811372,-5.62749847e-13 Z M493.340253,-5.62749847e-13 C503.023171,-5.62749847e-13 511.728914,2.07734807 519.457482,6.2320442 C527.18605,10.3867403 533.271187,16.1104972 537.712893,23.4033149 C542.154598,30.6961326 544.375451,38.8950276 544.375451,48 C544.375451,57.1049724 542.154598,65.3038674 537.712893,72.5966851 C533.271187,79.8895028 527.18605,85.6132597 519.457482,89.7679558 C511.728914,93.9226519 503.023171,96 493.340253,96 C483.657334,96 474.929383,93.9226519 467.156398,89.7679558 C459.383413,85.6132597 453.298276,79.8895028 448.900987,72.5966851 C444.503698,65.3038674 442.305054,57.1049724 442.305054,48 C442.305054,38.8950276 444.503698,30.6961326 448.900987,23.4033149 C453.298276,16.1104972 459.383413,10.3867403 467.156398,6.2320442 C474.929383,2.07734807 483.657334,-5.62749847e-13 493.340253,-5.62749847e-13 Z M23.3494372,-2.78532752e-13 L51.769038,64.5378151 L80.5889148,-2.78532752e-13 L102.070397,-2.78532752e-13 L61.6425143,90.3529412 L40.2944574,90.3529412 L-1.3580889e-13,-2.78532752e-13 L23.3494372,-2.78532752e-13 Z M379.9287,-2.78532752e-13 L379.9287,57.61269 L379.925848,58.216184 C379.825073,68.8283219 377.053741,76.81222 371.611853,82.1678784 C366.067288,87.6245869 357.923709,90.3529412 347.181115,90.3529412 C341.030113,90.3529412 335.377256,89.2319894 330.222543,86.9900859 C325.067831,84.7481824 320.844432,81.5545274 317.552347,77.409121 L317.552347,77.409121 L329.247913,63.7038995 L329.551113,64.098659 C334.416174,70.3496962 339.946306,73.4752148 346.141509,73.4752148 C354.718257,73.4752148 359.006632,68.5684071 359.006632,58.7547918 L359.006632,58.7547918 L359.006632,16.4970258 L327.03875,16.4970258 L327.03875,-2.78532752e-13 L379.9287,-2.78532752e-13 Z M209.811372,18.2983425 C204.303657,18.2983425 199.328946,19.558011 194.887241,22.0773481 C190.445535,24.5966851 186.958796,28.1104972 184.427024,32.6187845 C181.895251,37.1270718 180.629365,42.2541436 180.629365,48 C180.629365,53.7458564 181.895251,58.8729282 184.427024,63.3812155 C186.958796,67.8895028 190.445535,71.4033149 194.887241,73.9226519 C199.328946,76.441989 204.303657,77.7016575 209.811372,77.7016575 C215.319087,77.7016575 220.293797,76.441989 224.735503,73.9226519 C229.177209,71.4033149 232.663948,67.8895028 235.19572,63.3812155 C237.727492,58.8729282 238.993378,53.7458564 238.993378,48 C238.993378,42.2541436 237.727492,37.1270718 235.19572,32.6187845 C232.663948,28.1104972 229.177209,24.5966851 224.735503,22.0773481 C220.293797,19.558011 215.319087,18.2983425 209.811372,18.2983425 Z M493.340253,18.2983425 C487.832538,18.2983425 482.857827,19.558011 478.416122,22.0773481 C473.974416,24.5966851 470.487677,28.1104972 467.955905,32.6187845 C465.424132,37.1270718 464.158246,42.2541436 464.158246,48 C464.158246,53.7458564 465.424132,58.8729282 467.955905,63.3812155 C470.487677,67.8895028 473.974416,71.4033149 478.416122,73.9226519 C482.857827,76.441989 487.832538,77.7016575 493.340253,77.7016575 C498.847968,77.7016575 503.822678,76.441989 508.264384,73.9226519 C512.70609,71.4033149 516.192829,67.8895028 518.724601,63.3812155 C521.256373,58.8729282 522.522259,53.7458564 522.522259,48 C522.522259,42.2541436 521.256373,37.1270718 518.724601,32.6187845 C516.192829,28.1104972 512.70609,24.5966851 508.264384,22.0773481 C503.822678,19.558011 498.847968,18.2983425 493.340253,18.2983425 Z"
                      fill="#FFFFFF"
                      id="VOJO"
                    />
                  </g>
                </g>
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-AxjAm fPNbwy"
    >
      <div
        class="sc-AxirZ RVNDS"
      >
        <div
          class="sc-AxiKw dUPrhR"
        >
          <p
            class="sc-fznZeY iAWmBo"
          >
            Siga o Vojo nas redes sociais:
          </p>
          <div
            class="sc-AxhUy dtWeay"
          >
            <img
              src="https://s3.amazonaws.com/static.vojo.com.br/prd/images/facebook_branco.png"
            />
            <img
              src="https://s3.amazonaws.com/static.vojo.com.br/prd/images/instagram_branco.png"
            />
          </div>
        </div>
        <div
          class="sc-AxhCb jSTosF"
        >
          <div
            class="sc-AxgMl lxzaH"
          >
            <span
              class="sc-AxheI kkCIQt"
            >
              Termos de uso
            </span>
            <span
              class="sc-Axmtr bkBkiv"
            >
              Política de Privacidade
            </span>
          </div>
          <div
            class="sc-AxmLO ffTKFl"
          >
            <span
              class="sc-fzozJi cRdDET"
            >
              O VOJO é um produto da
              <span
                class="sc-fzpans cFTAwv"
              >
                Mind Lab
              </span>
              . Estamos localizados na Vila Mariana, em São Paulo – SP
              <span
                class="sc-fzoLsD kAsOlz"
              >
                <svg
                  fill="#FF5477"
                  height="12px"
                  stroke="#FF5477"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  width="12px"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"
                  />
                </svg>
              </span>
            </span>
          </div>
        </div>
        <div
          class="sc-fzplWN gdALIP"
        >
          <span
            class="sc-fznyAO iHyRZ"
          >
            2021
             – Vojo® – Todos os direitos reservados
          </span>
          <div
            class="sc-fznKkj jTYmss"
          >
            <svg
              class="Logo"
              height="16px"
              version="1.1"
              viewBox="0 0 618 96"
              width="103px"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            >
              <title>
                logo-vojo
              </title>
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="1"
              >
                <g>
                  <path
                    d="M601.267148,96 C605.982247,96 609.949831,94.3888756 613.169898,91.1666267 C616.389966,87.9443778 618,83.9165668 618,79.0831935 C618,74.1347399 616.389966,70.1356989 613.169898,67.0860705 C609.949831,64.0364421 605.982247,62.5116279 601.267148,62.5116279 C596.552049,62.5116279 592.584465,64.0364421 589.364398,67.0860705 C586.14433,70.1356989 584.534296,74.1347399 584.534296,79.0831935 C584.534296,83.9165668 586.14433,87.9443778 589.364398,91.1666267 C592.584465,94.3888756 596.552049,96 601.267148,96 Z"
                    fill="#FFFFFF"
                    id="Ponto"
                  />
                  <path
                    d="M209.811372,-5.62749847e-13 C219.49429,-5.62749847e-13 228.200033,2.07734807 235.928601,6.2320442 C243.657169,10.3867403 249.742306,16.1104972 254.184012,23.4033149 C258.625718,30.6961326 260.84657,38.8950276 260.84657,48 C260.84657,57.1049724 258.625718,65.3038674 254.184012,72.5966851 C249.742306,79.8895028 243.657169,85.6132597 235.928601,89.7679558 C228.200033,93.9226519 219.49429,96 209.811372,96 C200.128453,96 191.400502,93.9226519 183.627517,89.7679558 C175.854532,85.6132597 169.769395,79.8895028 165.372106,72.5966851 C160.974818,65.3038674 158.776173,57.1049724 158.776173,48 C158.776173,38.8950276 160.974818,30.6961326 165.372106,23.4033149 C169.769395,16.1104972 175.854532,10.3867403 183.627517,6.2320442 C191.400502,2.07734807 200.128453,-5.62749847e-13 209.811372,-5.62749847e-13 Z M493.340253,-5.62749847e-13 C503.023171,-5.62749847e-13 511.728914,2.07734807 519.457482,6.2320442 C527.18605,10.3867403 533.271187,16.1104972 537.712893,23.4033149 C542.154598,30.6961326 544.375451,38.8950276 544.375451,48 C544.375451,57.1049724 542.154598,65.3038674 537.712893,72.5966851 C533.271187,79.8895028 527.18605,85.6132597 519.457482,89.7679558 C511.728914,93.9226519 503.023171,96 493.340253,96 C483.657334,96 474.929383,93.9226519 467.156398,89.7679558 C459.383413,85.6132597 453.298276,79.8895028 448.900987,72.5966851 C444.503698,65.3038674 442.305054,57.1049724 442.305054,48 C442.305054,38.8950276 444.503698,30.6961326 448.900987,23.4033149 C453.298276,16.1104972 459.383413,10.3867403 467.156398,6.2320442 C474.929383,2.07734807 483.657334,-5.62749847e-13 493.340253,-5.62749847e-13 Z M23.3494372,-2.78532752e-13 L51.769038,64.5378151 L80.5889148,-2.78532752e-13 L102.070397,-2.78532752e-13 L61.6425143,90.3529412 L40.2944574,90.3529412 L-1.3580889e-13,-2.78532752e-13 L23.3494372,-2.78532752e-13 Z M379.9287,-2.78532752e-13 L379.9287,57.61269 L379.925848,58.216184 C379.825073,68.8283219 377.053741,76.81222 371.611853,82.1678784 C366.067288,87.6245869 357.923709,90.3529412 347.181115,90.3529412 C341.030113,90.3529412 335.377256,89.2319894 330.222543,86.9900859 C325.067831,84.7481824 320.844432,81.5545274 317.552347,77.409121 L317.552347,77.409121 L329.247913,63.7038995 L329.551113,64.098659 C334.416174,70.3496962 339.946306,73.4752148 346.141509,73.4752148 C354.718257,73.4752148 359.006632,68.5684071 359.006632,58.7547918 L359.006632,58.7547918 L359.006632,16.4970258 L327.03875,16.4970258 L327.03875,-2.78532752e-13 L379.9287,-2.78532752e-13 Z M209.811372,18.2983425 C204.303657,18.2983425 199.328946,19.558011 194.887241,22.0773481 C190.445535,24.5966851 186.958796,28.1104972 184.427024,32.6187845 C181.895251,37.1270718 180.629365,42.2541436 180.629365,48 C180.629365,53.7458564 181.895251,58.8729282 184.427024,63.3812155 C186.958796,67.8895028 190.445535,71.4033149 194.887241,73.9226519 C199.328946,76.441989 204.303657,77.7016575 209.811372,77.7016575 C215.319087,77.7016575 220.293797,76.441989 224.735503,73.9226519 C229.177209,71.4033149 232.663948,67.8895028 235.19572,63.3812155 C237.727492,58.8729282 238.993378,53.7458564 238.993378,48 C238.993378,42.2541436 237.727492,37.1270718 235.19572,32.6187845 C232.663948,28.1104972 229.177209,24.5966851 224.735503,22.0773481 C220.293797,19.558011 215.319087,18.2983425 209.811372,18.2983425 Z M493.340253,18.2983425 C487.832538,18.2983425 482.857827,19.558011 478.416122,22.0773481 C473.974416,24.5966851 470.487677,28.1104972 467.955905,32.6187845 C465.424132,37.1270718 464.158246,42.2541436 464.158246,48 C464.158246,53.7458564 465.424132,58.8729282 467.955905,63.3812155 C470.487677,67.8895028 473.974416,71.4033149 478.416122,73.9226519 C482.857827,76.441989 487.832538,77.7016575 493.340253,77.7016575 C498.847968,77.7016575 503.822678,76.441989 508.264384,73.9226519 C512.70609,71.4033149 516.192829,67.8895028 518.724601,63.3812155 C521.256373,58.8729282 522.522259,53.7458564 522.522259,48 C522.522259,42.2541436 521.256373,37.1270718 518.724601,32.6187845 C516.192829,28.1104972 512.70609,24.5966851 508.264384,22.0773481 C503.822678,19.558011 498.847968,18.2983425 493.340253,18.2983425 Z"
                    fill="#FFFFFF"
                    id="VOJO"
                  />
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
